infinite-list/infinite-list.tag

```diff
@@ -0,0 +1,25 @@
+<infinite-list onscroll={ onscroll }>
+
+  <h3>{ title }</h3>
+  
+  <list-item each={ items_to_render }></list-item>
+
+  this.title           = opts.title
+  this.items           = opts.items || []
+  this.items_to_render = this.items.slice(0, 250)
+  
+  onscroll(e) {
+   
+    var itemheight  = 72  // Height of item
```

Comment:
I'm not sure, I couldn't find the style which define this height. I wonder it depends on environment...?